### DISCUSSION TITLE
PHP $_GET and $_POST are always empty.

### DISCUSSION THREAD

**Discussion by timoabdou** [yandeu/five-server#122] [Help / Questions] [answered]
Thanks for the amazing extension.
I have an issue when passing variables using GET or POST methods (in both links and forms), the $_GET & $_POST arrays are always empty, is this a limitation  of the five server, or am I doing some thing wrong?
page A.php content:

```
<a href="process.php?id=1">Link</a>
    <form action="process.php" method="post">
        <input type="hidden" name="id" id="" value="2">
        <input type="submit" value="Form">
    </form>
```

page B.php content:
```
<?php
var_dump($_GET);
var_dump($_POST);
```

**Comment by yandeu** [comment, marked as answer]
Thanks :)

Yes, this is a limitation. Five Server does just render PHP but doesn't include a real PHP Server.

See: https://youtu.be/Qk3WRkzUB88?si=q-t0gb5FMq93JF5i

**Reply by timoabdou** [reply]
Thank you for the clarification.
